### DISCUSSION TITLE
LoRA in SD 1.5 usable ?

### DISCUSSION THREAD

**Discussion by Coolzeroalpha** [lllyasviel/stable-diffusion-webui-forge#449] [Q&A] [answered]
Hello, 

is there any way to use SD 1.5 LoRA's into WebUI Forge ? 

It seems that WebUI Forge only supports SDXL, and It's not possible for the same project to use 2 software applications because the results are too different.

Thanks :)

**Comment by freecoderwaifu** [comment, marked as answer]
1.5 and SDXL Loras work, 1.5 is fully supported. Keep in mind you can't use 1.5 Loras with SDXL checkpoints and viceversa, same as in any web UI.

**Reply by VeteranXT** [reply]
That wound not make any sense if they worked!.

**Reply by freecoderwaifu** [reply]
Yeah but newcomers in general might not be aware.

**Reply by Coolzeroalpha** [reply]
Oh ok, I didn't know that. It's works !!! haha thank you
Because when I use Fooocus I'm able to choose SDXL model and an SD 1.5 LoRA. But maybe that why sometime I can't have good result, I didn't pay attention about which combinaison I used when I had bad result

**Reply by Coolzeroalpha** [reply]
Yes I'm just start generative image by AI 2 weeks ago, too many things to learn and too many basics I don't have yet
Thanks :)
